go.mod

```diff
@@ -4,50 +4,59 @@ go 1.20
 
 require (
 	github.com/docker/go v1.5.1-1
-	github.com/go-openapi/runtime v0.26.0
-	github.com/google/go-cmp v0.5.9
-	github.com/google/trillian v1.5.2 // indirect
+	github.com/go-openapi/runtime v0.26.2
+	github.com/google/go-cmp v0.6.0
+	github.com/google/trillian v1.5.3 // indirect
 	github.com/in-toto/in-toto-golang v0.9.0
-	github.com/secure-systems-lab/go-securesystemslib v0.7.0
-	github.com/sigstore/rekor v1.2.2
-	github.com/sigstore/sigstore v1.7.2
+	github.com/secure-systems-lab/go-securesystemslib v0.8.0
+	github.com/sigstore/rekor v1.3.4
+	github.com/sigstore/sigstore v1.8.0
 )
 
 require (
-	github.com/go-openapi/strfmt v0.21.7
-	github.com/go-openapi/swag v0.22.4
-	github.com/google/go-containerregistry v0.16.1
+	github.com/go-openapi/strfmt v0.22.0
+	github.com/go-openapi/swag v0.22.7
+	github.com/google/go-containerregistry v0.17.0
 	github.com/gorilla/mux v1.8.0
 	github.com/sigstore/cosign/v2 v2.2.0
+	github.com/sigstore/sigstore-go v0.0.0-20240108223800-a3df13b8ba29
 	github.com/slsa-framework/slsa-github-generator v1.4.0
-	github.com/spf13/cobra v1.7.0
-	golang.org/x/mod v0.13.0
-	sigs.k8s.io/release-utils v0.7.4
+	github.com/spf13/cobra v1.8.0
+	golang.org/x/mod v0.14.0
+	sigs.k8s.io/release-utils v0.7.7
 )
 
+// use the pending PR #41 branch tuf-client-2
+replace github.com/sigstore/sigstore-go => github.com/sigstore/sigstore-go v0.0.0-20231222133331-d489b534902f
```

Comment:
note to you: don't forget to replace before merging

Reply:
It's now using sigstore/sigstore-go@0.2.0